src/oop_ext/foundation/_tests/test_handle_exception.py

```diff
@@ -0,0 +1,59 @@
+
+import sys
+
+import pytest
+
+from oop_ext.foundation import handle_exception
+
+
+@pytest.fixture
+def captured_handled_exceptions():
+    """
+    Captures the exceptions using handle_exception module.
```

Comment:
@kfasolin any feeling towards this? My feeling is that this is weird to have in `oop_ext` and we should consider moving it back to `ben10`... (not a critic of your job here of course @lincrosenbach)

Reply:
I agree, actually all code related with exceptions and reraise I think should be kept on ben10.

Reply:
Oh specially `Reraise`... it is absolutely useless in Python 3, and even breaks in case exceptions receive more than one parameter.

Reply:
Well then we have to decide what to do with the exception handling on interface Check() method implementation.